src/OpenTelemetry/Metrics/Exemplar/SimpleFixedSizeExemplarReservoir.cs

```diff
@@ -42,15 +42,15 @@ protected override void OnCollected()
     private void Offer<T>(in ExemplarMeasurement<T> measurement)
         where T : struct
     {
-        var measurementNumber = this.measurementsSeen++;
+        var measurementNumber = Interlocked.Increment(ref this.measurementsSeen) - 1;
```

Comment:
Could we initialize `this.measurementsSeen` to `-1` ?

Reply:
Sure! I made this change: https://github.com/open-telemetry/opentelemetry-dotnet/pull/5465/commits/f8587b5749e614fc944859320e915ef9d24496aa

Probably a bit more than you were expecting. I renamed things because I didn't like having it named "measurementsSeen" with a value of `-1`. How do you see -1 of something? 🤣